.Xresources

```diff
@@ -11,6 +11,9 @@ st.borderless:        0
 ! --- The following options options can be reloaded via USR1 signal. ---
```

Comment:
note so I don't forget: we'll also want to mention `font2` in the patches list in the readme

Reply:
I'll add it.